doc/man1/openssl-pkcs12.pod.in

```diff
@@ -362,7 +362,7 @@ to be needed to use MAC iterations counts but they are now used by default.
 
 =item B<-nomac>
 
-Don't attempt to provide the MAC integrity. This can be useful with FIPS
+Don't attempt to provide the MAC integrity. This can be useful with the FIPS
```

Comment:
Now that I reread this: `s/Don't/Do not/` we're supposed to avoid apostrophes.

Reply:
Fixed.

Reply:
> Now that I reread this: s/Don't/Do not/ we're supposed to avoid apostrophes.

Is that style/policy now?  @levitte disagreed in some PR of mine when I proposed making that change to find-doc-nits.